ext/standard/basic_functions.stub.php

```diff
@@ -312,16 +312,16 @@ function php_strip_whitespace(string $filename): string {}
 
 function highlight_string(string $string, bool $return = false): string|bool {}
 
-function ini_get(string $name): string|false {}
+function ini_get(string $setting): string|false {}
```

Comment:
We also discussed these, and decided to use `$name`, but I'd like to ask for reconsideration :) I think `$setting` (`$config` or even `$option`) is more self explanatory than `$name`.

Reply:
I don't like `$setting` (we also don't use this term anywhere else yet), but I'd be okay with `$option` or `$config` (based on existing usage, `$option` would be preferred, `$config` is also commonly used but in a more general sense, e.g. a config object or a config file).